include/library/spdm_transport_tcp_lib.h

```diff
@@ -0,0 +1,149 @@
+/**
+ *  Copyright Notice:
+ *  Copyright 2025 DMTF. All rights reserved.
+ *  License: BSD 3-Clause License. For full text see link: https://github.com/DMTF/libspdm/blob/main/LICENSE.md
+ **/
+
+#ifndef SPDM_TCP_TRANSPORT_LIB_H
+#define SPDM_TCP_TRANSPORT_LIB_H
+
+#include "library/spdm_common_lib.h"
+
+#define LIBSPDM_TCP_ALIGNMENT 4
+
+#define LIBSPDM_TCP_SEQUENCE_NUMBER_COUNT 0
```

Comment:
These are defined in the specification and should go in `industry_standard/`.

Reply:
Done.